docs/content/en/docs/pipeline-stages/filesync.md

```diff
@@ -91,12 +93,18 @@ Another thing the Buildpacks have to do is support the `GOOGLE_DEVMODE` environm
 set it to `1` when running `skaffold dev` with sync configured to `auto: {}`. The Buildpacks can then use that
 signal to change the way the application is built so that it reloads the changes or rebuilds the app on each change.
 
+#### Jib
+
+Jib integration with skaffold allows for zero config `auto` sync. In this mode, Jib will sync your `classes`, `resources` and `extraDirectories` to a remote container as changes are made. It can only be used with Jib in default build mode (exploded) for jar style applications. It was primarily designed around [Spring Boot Developer Tools](https://docs.spring.io/spring-boot/docs/current/reference/html/using-spring-boot.html#using-boot-devtools), but can work with any embedded server that can reload/restart.
```

Comment:
How about this?

```suggestion
Jib integration with Skaffold allows for zero-config `auto` sync. In this mode, Jib will sync your class files, resource files, and Jib's "extra directories" files to a remote container as changes are made. It can only be used with Jib in the default build mode (exploded) for non-WAR applications. It was primarily designed around [Spring Boot Developer Tools](https://docs.spring.io/spring-boot/docs/current/reference/html/using-spring-boot.html#using-boot-devtools), but can work with any embedded server that can reload/restart.
```
skaffold -> Skaffold
zero config -> zero-config
`classes`, `resources` and `extraDirectories` --> class files, resource files, and Jib's "extra directories" files
default build mode -> the default build mode
jar style -> non-WAR